client/core/core.go

```diff
@@ -2717,6 +2717,11 @@ func (c *Core) MaxBuy(host string, base, quote uint32, rate uint64) (*MaxOrderEs
 	}
 
 	quoteLotEst := calc.BaseToQuote(rate, baseAsset.LotSize)
+
+	if quoteLotEst == 0 {
+		return nil, errors.New("cannot divide by lot size zero")
+	}
+
```

Comment:
Can go before `c.dex` and `c.feeSuggestion`, right after getting `baseAsset` from `marketWallets`

Reply:
Ok.